packages/ember-metal/lib/weak_map.js

```diff
@@ -1,7 +1,10 @@
+import { typeOf } from 'ember-runtime/utils';
+import { assert } from 'ember-metal/debug';
 import { GUID_KEY } from 'ember-metal/utils';
 import { meta } from 'ember-metal/meta';
 
 var id = 0;
+const UNDEFINED = function() {};
```

Comment:
lets do `function UNDEFINED() {}` that way it is named nicely in most modern debuggers.